plugin/trino-oracle/src/test/java/io/trino/plugin/oracle/BaseOracleFailureRecoveryTest.java

```diff
@@ -59,4 +61,12 @@ protected QueryRunner createQueryRunner(
                             "exchange.base-directories", System.getProperty("java.io.tmpdir") + "/trino-local-file-system-exchange-manager"));
                 });
     }
+
+    @Override
+    @Flaky(issue = "16277", match = "There should be no remaining tmp_trino tables that are queryable")
```

Comment:
We usually put an issue URL in this field. 
```suggestion
    @Flaky(issue = "https://github.com/trinodb/trino/issues/16277", match = "There should be no remaining tmp_trino tables that are queryable")
```